### ISSUE TITLE
Unclear How Mouse Buttons Work

### ISSUE THREAD

**Issue by domiluci** [jpconstantineau/BlueMicro_HID_Arduino_Library#9]
This may be somewhat of a trivial question, but after combing through the examples and code it's unclear how to use `mouseButtonPress` and `mouseReport` for both the Left Click and Right Click of the mouse. I see `buttons` is a `uint8_t`, so I'm assuming it's a hex value like the keyboard keys. I just can't find the values.

Any help you can give would be great! Thank you!

**Comment by jpconstantineau** [comment]
Indeed, I could add more examples and documentation on this.  Since the implementation uses Tiny USB under the hood, you can have a look at these examples for inspiration. 

The values would be buried in the Tiny USB Arduino library implementation.
